packages/grafana-ui/src/components/Slider/_Slider.scss

```
@@ -1 +1 @@
-@import '../../node_modules/rc-slider/assets/index.css';
+@import '../../packages/grafana-ui/node_modules/rc-slider/assets/index.css';
```

```suggestion
@import 'rc-slider/assets/index.css';
```
https://sass-lang.com/documentation/at-rules/import